astroid/builder.py

```diff
@@ -190,8 +179,18 @@ def _data_build(self, data, modname, path):
             )
         builder = rebuilder.TreeRebuilder(self._manager, parser_module)
         module = builder.visit_module(node, modname, node_file, package)
-        module._import_from_nodes = builder._import_from_nodes
-        module._delayed_assattr = builder._delayed_assattr
+
+        self._manager.cache_module(module)
```

Comment:
Shouldn't the `cache_module` stay in `_post_build`? If we ever do something different in `_data_build` after this line or we add a method between `_data_build` and `_post_build` it won't be cached?

Reply:
I think it wouldn't matter. The `module` object reference is cached, ie. if the object changes, those changes are also reflected in the cache.